pipelines/rj_cor__precipitacao_alertario_sftp/utils.py

```
@@ -240,6 +240,7 @@ def transform_meteorological_dataframe(dfr: pd.DataFrame) -> pd.DataFrame:
     ]
 
     dfr = dfr.drop_duplicates(subset=["id_estacao", "data_medicao"], keep="first")
+    dfr.drop(dfr[dfr['data_medicao'] > datetime.now().strftime('%Y-%m-%d %H:%M:%S')].index, inplace=True)
```

Ruff (0.16.3)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)